third_party/EGL-Registry/CMakeLists.txt

```diff
@@ -16,6 +16,13 @@ cmake_minimum_required(VERSION 3.13)
 
 project(egl_headers C)
 
+if(NOT IS_DIRECTORY ${SHADERTRAP_EGL_REGISTRY_REPO_DIR})
```

Comment:
Done. I found that I had to keep `project` for glad; otherwise I got errors from CMake - seems like it needs to know that the language is C if a non-interface library is being defined.